tests/HomeworkCalculatorTesting.spec.js

```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require('@playwright/test');
+/*const { BasicCalculatorPage } = require('../pages-Calculator/basicCalculatorPage');
+
+
+test.describe('', () => {
+    let page;
+    
+    test.beforeAll(async ({ browser }) => {
+      page = await browser.newPage();
+      startPage = new BasicCalculatorPage(page);
+
+        });
+   test.beforeEach(async () => {
+      await startPage.goto();
+    });
+  });*/
+       
+ 
+test('Testing page to be opened', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
```

Comment:
I wonder why the main describe is commented. This line could be added to beforeEach hook in order to prevent from duplicating the code.